### PR TITLE
[WIP] Improvements 

### PR DESCRIPTION
**Pull request by mosn**
- [x] All links must work
- [x] Image optimization 
- [x] Assets path correct
- [ ] Get feedback on new asset folder
Optional 
- [ ] Match images with UI 

Fixes kubeflow/kubeflow#4025
Checking some other places for similar issues. will try to fix here

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1156)

### CODE REVIEW

**Comment by mosn** [comment]
Ready to merge

File: content/docs/other-guides/virtual-dev/getting-started-minikube.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ chmod +x setup-minikube.sh
 **KUBEFLOW_TAG** is a tag corresponding to the version to checkout such as {{% kf-latest-version %}} or `master`.
 
 The script asks for some config input as shown below:
-![ConfigInputs](../ConfigInputs.png)
+![ConfigInputs](/content/docs/started/ConfigInputs.png)
```
Comment (sarahmaddox) [inline content/docs/other-guides/virtual-dev/getting-started-minikube.md]: Hallo @mosn thanks for taking this on!

We usually put the images in the `images` directory, tho that's not essential:
https://github.com/kubeflow/website/tree/master/content/docs/images

The path to the image should start at `/docs/` (not at `/content/`).

While you're here, please would you fix the styling of the image too? (There's a bit about images in the [README](https://github.com/kubeflow/website/blob/master/README.md) though that could certainly be expanded.)

Here's an example of an image with styling and path:

```
<img src="/docs/images/consent-screen.png" 
  alt="OAuth consent screen"
  class="mt-3 mb-3 p-3 border border-info rounded">
```

The above example comes from this page:
https://github.com/kubeflow/website/tree/master/content/docs/images

**Comment by sarahmaddox** [comment]
Friendly ping @mosn please would you take a look at the review comments on this PR?

**Comment by mosn** [comment]
Sure thing, Traveling currently but have the whole day in transit to finalize this.

**Comment by mosn** [comment]
@sarahmaddox  I am trying to go over all images to make them looks better. I can think of two, 
1) setting up  Kubeflow Component library in  [Storybook](https://storybook.js.org/) Library 
2) or a simple Sketch file with UI Elements. 
Component library seems  the best approach in long run but Sketch file is a quick start.

**Comment by mosn** [comment]
Since this is the feedback phase, I skip through the documentation and presentation for the changes. 

1.  organize all images in one location `content/docs/images/azure/`
2. mimic folder structure in content/docs/ inside the `images` 
3. Pending add a git pre-commit hook to move the new images to where it belongs. that way we can just have the .md file .png in one location and pre-commit should take care of putting it in the right folder

Any thoughts?

**Pull request closed without merging**